components/App.js

```diff
@@ -0,0 +1,99 @@
+import TodoInput from "./Input";
+import TodoList from "./TodoList";
+import Controller from "./Controller";
+import filterTodos from "../utils/filterTodos";
+
+export default function App({ $target }) {
+  this.state = {
+    todos: [],
+    filter: "",
+  };
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  this.render = () => {
+    const filteredTodos = filterTodos(this.state.filter, this.state.todos);
+    todoList.setState(filteredTodos);
+    controller.setState(filteredTodos.length);
+  };
+
+  const setTodos = (newTodoList) => {
+    this.setState({
+      ...this.state,
+      todos: newTodoList,
+    });
+  };
+
+  const onAddTodo = (newTodoContent) => {
+    const newTodo = {
+      id: new Date().getTime().toString(),
```

Comment:
아래처럼 사용해보셔도 좋을거 같네요~

```suggestion
      id: Date().now().toString(),
```